bluesky_widgets/qt/run_tree.py

```diff
@@ -0,0 +1,195 @@
+from qtpy import QtCore
+from qtpy.QtCore import (
+    QAbstractItemModel,
+    QModelIndex,
+    QTimer,
+    Qt,
+)
+from qtpy.QtWidgets import QAbstractItemView, QHeaderView, QTreeView
+
+from databroker import catalog
+from databroker.core import BlueskyEventStream
+
+class RunTree(object):
```

Comment:
Subclassing from `object` was necessary in Python 2 to opt in to so-called "new style" classes, but in Python 3-only codebases it's no longer necessary. It's a no-op.

Reply:
Thanks for the explanation, I will fix this up, I was used to using this a lot but I think we were spanning 2 and 3.